spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySource.java

```diff
@@ -249,6 +272,56 @@ private Object createFeature(ConfigurationSetting item) throws IOException {
                 } else if (!featureItem.getEnabled()) {
                     return false;
                 }
+                mapper.configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true);
+                for (int filter = 0; filter < feature.getEnabledFor().size(); filter++) {
+                    if (feature.getEnabledFor().get(filter).getName().equals(TARGETING_FILTER)) {
+                        FeatureFilterEvaluationContext context = feature.getEnabledFor().get(filter);
```

Comment:
It's not good practice to use as an internal variable name the same as some field class name.